File: POO_Intro/POO_Uber/Javascript/car.js
function Car(license, driver) {
    this.id;
    this.license = license;
    this.driver = driver;
    this.passengers;
}

Car.prototype.printDataCar = function() {
    console.log(this.license)
    console.log(this.driver.name)
    console.log(this.passengers)
}

File: POO_Intro/POO_Uber/Java/Car.java
class Car{
    Integer id;
    String license;
    Account driver;
    private Integer passenger;

    public Car(String license, Account driver){
        this.license = license;
        this.driver = driver;
    }

    void printDataCar () {
        System.out.println("License: " + license + ", Driver: " + driver.name );
        System.out.println("Pasajeros: " + passenger);
    }

    public Integer getPassenger(){
        return passenger;
    }

    public void setPassenger(Integer passenger){
        if(passenger <= 4){
            this.passenger = passenger;
        }else{
            System.out.println("Debe ser menor de 4 pax");
        }
    }
}